Allowed creation of Experience entity in database with proper

GabrielRodriguez66 opened the pull request:
field validators. Implemented the corresponding form for experience
registration, added experience registration route and added the option
in the nav bar.

GabrielRodriguez66 committed:
Allowed creation of Experience entity in database with proper

field validators. Implemented the corresponding form for experience
registration, added experience registration route and added the option
in the nav bar.

Joshuacruzc commented on .idea/ResumeBuilder_.iml:
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Add .idea/ directory to .gitignore file.

Joshuacruzc commented on resumebuilder/routes.py:
```diff
@@ -1,15 +1,15 @@
     return render_template('index.html')
 
-@app.route('/register', methods=['GET', 'POST'])
+@app.route('/', methods=['GET', 'POST'])
```
revert

GabrielRodriguez66 committed:
Reverted a deletion and added .gitignore

GabrielRodriguez66 commented on .gitignore:
```diff
@@ -0,0 +1 @@
+.idea/
```
Añadí el directorio de .idea/

Joshuacruzc replied:
slash is not necessary.  
place comments in Discussions and close them if you did what was asked for.

GabrielRodriguez66 commented on resumebuilder/routes.py:
```diff
@@ -9,7 +10,7 @@
     return render_template('index.html')
 
-@app.route('/', methods=['GET', 'POST'])
+@app.route('/register', methods=['GET', 'POST'])
```
Revertí el cambio erróneo

Joshuacruzc commented on resumebuilder/templates/layout.html:
```diff
@@ -23,9 +23,12 @@
   <ul class="navbar-nav ml-auto">
-      {%  if current_user.is_authenticated %}
+      {% if current_user.is_authenticated %}
+          <li>
```
give <li> tag class of nav-item and <a> tag class of nav-link for consistency and css styling.

Joshuacruzc commented on resumebuilder/routes.py:
```diff
@@ -1,11 +1,12 @@
 
 
-@app.route('/', methods=['GET', 'POST'])
+@app.route('/index', methods=['GET', 'POST'])
```
revert

Joshuacruzc commented on resumebuilder/routes.py:
```diff
@@ -43,4 +44,31 @@ def login():
\ No newline at end of file
+    return redirect(url_for('index'))
+
+
```
include @login_required decorator from flask_login to prevent page being accessed by unauthenticated users.

GabrielRodriguez66 committed:
Included @login_required decorator in Experience Registration route

and Logout route from flask_login to prevent page being accessed
by unauthenticated users... also, fixed the .gitignore

GabrielRodriguez66 committed:
Added .idea files individually to .gitignore (testing if it works)
